assets/src/components/app.tsx

```diff
@@ -24,6 +28,9 @@ const App = (): ReactElement<HTMLDivElement> => {
         <BrowserRoute exact={true} path="/settings" component={SettingsPage} />
         <BrowserRoute path="/about" component={AboutPage} />
         <BrowserRoute exact={true} path="/search" component={SearchPage} />
+        {connectionStatus === ConnectionStatus.Disconnected ? (
```

Comment:
It's also possible to do:

    {connectionStatus === ConnectionStatus.Disconnected && ( <DisconnectedModal /> )}

if you wanted to avoid the `null`.

Reply:
eh, I don't mind the `null` since it's explicitly saying "don't draw anything".